### PR TITLE
{math}[system/system] MATLAB v2022a, MATLAB v2022a-r3

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

I am not insisting on having the `Update-3` version but as I am using it in PR #16201 I thought I might put it up as well.

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@sassy-crick why remove the `license_server` comment? (comparing to the existing easyconfigs)

```diff
 # Use EB_MATLAB_LICENSE_SERVER and EB_MATLAB_LICENSE_SERVER_PORT environment variables or
 # uncomment and modify the following variables for installation with floating license server
 # license_file = 'my-license-file'
-# license_server = 'my-license-server'
-# license_server_port = '1234'
+# license_server_port = 'XXXXX'
 
 moduleclass = 'math'
```

also, can you submit a test report?

**Comment by sassy-crick** [comment]
Apologies for my slow reply! I have uploaded the two test reports [here](https://gist.github.com/sassy-crick/4b95961af559a15e0ff365655cdbac60) and I have addressed the changed

File: easybuild/easyconfigs/m/MATLAB/MATLAB-2022a_Update_3.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+version = '2022a_Update_3'
+
+homepage = 'https://www.mathworks.com/products/matlab'
+description = """MATLAB is a high-level language and interactive environment
+ that enables you to perform computationally intensive tasks faster than with
+ traditional programming languages such as C, C++, and Fortran."""
+
+toolchain = SYSTEM
+
+sources = ['R%(version)s_Linux.iso']
```
Comment (lexming) [inline easybuild/easyconfigs/m/MATLAB/MATLAB-2022a_Update_3.eb:11]: Then the sources can be written as
```suggestion
sources = ['R%s_Update_%s_Linux.iso' % (version, _update)]
```

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
login2.cerberus.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 3.6.8
See https://gist.github.com/79b4461df4ba04443d739cfae6f98e87 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @sassy-crick !

**Pull request merged**